hack/generate-proto.sh

```diff
@@ -33,7 +33,6 @@ APIMACHINERY_PKGS=(
     k8s.io/api/core/v1
 )
 go-to-protobuf \
-    --logtostderr \
```

Comment:
accident?

Reply:
The newer codegen utilities removed this flag as a result of the switch from glog to klog, so I needed to remove it.